lib/ansible/modules/crypto/openssl_dhparam.py

```diff
@@ -114,7 +114,7 @@ def generate(self, module):
             module.add_cleanup_file(tmpsrc)  # Ansible will delete the file on exit
             # openssl dhparam -out <path> <bits>
             command = [self.openssl_bin, 'dhparam', '-out', tmpsrc, str(self.size)]
-            rc, _, err = module.run_command(command, check_rc=False)
+            rc, _out, err = module.run_command(command, check_rc=False)
```

Comment:
Do you plan to use `_out`?

Reply:
No, ansible doesn't allow `_` as a variable name for reasons unknown.

Reply:
Are you sure? `_` has been in use in the letsencrypt module (see https://github.com/ansible/ansible/blob/devel/lib/ansible/modules/web_infrastructure/letsencrypt.py) for a long time, and so far nobody complained. (Maybe it's not alloewd in `jinja2` templates?)

Reply:
I think it's a new thing in the CI checks. It's the reason https://app.shippable.com/github/ansible/ansible/runs/51995/summary failed though.

Reply:
See also https://github.com/ansible/ansible/pull/32620#issuecomment-346000767

Reply:
Very interesting. I haven't seen this happening for all my `letsencrypt` PRs over the last few weeks. I'll investigate a bit, this made me very curious :)

Reply:
I found something: https://github.com/ansible/ansible/blob/devel/docs/docsite/rst/dev_guide/testing/sanity/no-underscore-variable.rst
Interesting... So the recommended name is `dummy`. I'll create another PR for `letsencrypt`...

Reply:
I like `_out` slightly better, it at least still documents that it's the stdout of the command while the `_`  indicates it's not used. Also it would again reset all the shipits to 0 :disappointed:

Reply:
I regularly check it anyways and you need one from the core team, you can ping people on IRC in the #ansible-devel channel.

I like neither _out nor dummy, but `dummy` seems to be the one that the ansible project prefers...

Reply:
Ok, ok, done it. I think core team will only review it after it's marked `shipit`.